collection-scripts/gather_sub

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+DIR_NAME=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+# shellcheck disable=SC1091
+source "${DIR_NAME}/common.sh"
+
+IFS=$'\n'
+
+mkdir -p "${BASE_COLLECTION_PATH}"
+/usr/bin/oc get packagemanifest -o yaml >>"${BASE_COLLECTION_PATH}/packagemanifests"
+
+mkdir -p "${BASE_COLLECTION_PATH}/subscriptions/"
+for service in $(oc get subscriptions -o=custom-columns=NAME:.metadata.name --no-headers)
+do
+    /usr/bin/oc get subscription "$service" -o yaml >> "${BASE_COLLECTION_PATH}/subscriptions/$service"
+done
+
+mkdir -p "${BASE_COLLECTION_PATH}/installplans/"
+for service in $(oc get installplan -o=custom-columns=NAME:.metadata.name --no-headers)
+do
+    /usr/bin/oc get installplan "$service" -o yaml >> "${BASE_COLLECTION_PATH}/installplans/$service"
```

Comment:
@fmount with current image InstallPlans and subscriptions folder are empty is it expected?
- https://logserver.rdoproject.org/59/359/4ae53455e0ca1f00282fdb25d5469e7dd19fac24/github-check/ci-framework-crc-podified-edpm-baremetal/04b056b/ci-framework-data/logs/quay-io-fpantano-must-gather-sha256-d9b2936e6e683c47ed67a34a706ad2e2f02a3d120ae71111968868a31206639b/installplans/
- https://logserver.rdoproject.org/59/359/4ae53455e0ca1f00282fdb25d5469e7dd19fac24/github-check/ci-framework-crc-podified-edpm-baremetal/04b056b/ci-framework-data/logs/quay-io-fpantano-must-gather-sha256-d9b2936e6e683c47ed67a34a706ad2e2f02a3d120ae71111968868a31206639b/subscriptions/

Reply:
Fixed in the last iteration (v63 image): both `subscriptions` and `installplans` are `namespace scoped` resources, hence it's expected to see the global directory empty.
In my last iteration I kept both `CSVs` and `pkgmanifests` as global resources (they don't depend on the passed namespace), while I moved `subs` and `installplans` within the created namespace tree.
If you re-kick the edpm CI you will see the resources appearing within the `openstack-operators` namespace:

```
[stack@osp-storage-05 glance_operator_test]$ ls must-gather.local.8439580022763696998/quay-io-fpantano-must-gather-sha256-274d7a67c976603d2d9f5b0d5d3a98d27c6df2bb39a20411c48fd1b514591df0/namespaces/openstack-operators/installplans/
install-crkt7
[stack@osp-storage-05 glance_operator_test]$ ls must-gather.local.8439580022763696998/quay-io-fpantano-must-gather-sha256-274d7a67c976603d2d9f5b0d5d3a98d27c6df2bb39a20411c48fd1b514591df0/namespaces/openstack-operators/subscriptions/
cinder-operator-alpha-openstack-operator-index-openstack-operators     mariadb-operator-alpha-openstack-operator-index-openstack-operators
cluster-operator-alpha-openstack-operator-index-openstack-operators    neutron-operator-alpha-openstack-operator-index-openstack-operators
dataplane-operator-alpha-openstack-operator-index-openstack-operators  nova-operator-alpha-openstack-operator-index-openstack-operators
glance-operator-alpha-openstack-operator-index-openstack-operators     openstack-ansibleee-operator-alpha-openstack-operator-index-openstack-operators
horizon-operator-alpha-openstack-operator-index-openstack-operators    openstack-baremetal-operator-alpha-openstack-operator-index-openstack-operators
infra-operator-alpha-openstack-operator-index-openstack-operators      openstack-operator
ironic-operator-alpha-openstack-operator-index-openstack-operators     ovn-operator-alpha-openstack-operator-index-openstack-operators
keystone-operator-alpha-openstack-operator-index-openstack-operators   placement-operator-alpha-openstack-operator-index-openstack-operators
manila-operator-alpha-openstack-operator-index-openstack-operators     telemetry-operator-alpha-openstack-operator-index-openstack-operators
```

Reply:
updating the ci-framework patch to latest image.

Reply:
Here are the logs of v63: https://logserver.rdoproject.org/59/359/546f3cd7558d51a31ee1311f948102b0638b7319/github-check/ci-framework-crc-podified-edpm-deployment/34e6022/ci-framework-data/logs/quay-io-fpantano-must-gather-sha256-274d7a67c976603d2d9f5b0d5d3a98d27c6df2bb39a20411c48fd1b514591df0/

Thank you for fixing it. Logs are available here:
- https://logserver.rdoproject.org/59/359/546f3cd7558d51a31ee1311f948102b0638b7319/github-check/ci-framework-crc-podified-edpm-deployment/34e6022/ci-framework-data/logs/quay-io-fpantano-must-gather-sha256-274d7a67c976603d2d9f5b0d5d3a98d27c6df2bb39a20411c48fd1b514591df0/namespaces/openstack-operators/installplans/
- https://logserver.rdoproject.org/59/359/546f3cd7558d51a31ee1311f948102b0638b7319/github-check/ci-framework-crc-podified-edpm-deployment/34e6022/ci-framework-data/logs/quay-io-fpantano-must-gather-sha256-274d7a67c976603d2d9f5b0d5d3a98d27c6df2bb39a20411c48fd1b514591df0/namespaces/openstack-operators/subscriptions/

Now it have everything needed from ci-framework side.

Reply:
ack thanks for the feedback.
@raukadah in my last iteration I also added `gather_networks`, so we can get info about what nmstate and metallb configure:

```
[stack@osp-storage-04 quay-io-fpantano-must-gather-sha256-e222522cba3ca0bec0ba7c8c99d539fa15f53c6ad0dfd91c74804aed8320f78e]$ tree network/
network/
├── ipaddresspools
│   ├── ctlplane.log
│   ├── internalapi.log
│   ├── storage.log
│   └── tenant.log
├── l2advertisement.log
└── nncp
    ├── enp6s0-crc-8cf2w-master-0.log
    └── enp6s0-crc-chf7b-master-0.log

```